src/lightning_app/cli/cmd_clusters.py

```diff
@@ -0,0 +1,213 @@
+import json
+import re
+import time
+from datetime import datetime
+
+import arrow
+import click
+from lightning_cloud.openapi import (
+    V1AWSClusterDriverSpec,
+    V1ClusterDriver,
+    V1ClusterPerformanceProfile,
+    V1ClusterSpec,
+    V1CreateClusterRequest,
+    V1InstanceSpec,
+    V1KubernetesClusterDriver,
+)
+from lightning_cloud.openapi.models import Externalv1Cluster, V1ClusterState, V1ClusterType
+from rich.console import Console
+from rich.table import Table
+from rich.text import Text
+
+from lightning_app.cli.core import Formatable
+from lightning_app.utilities.network import LightningClient
+from lightning_app.utilities.openapi import create_openapi_object, string2dict
+
+CLUSTER_STATE_CHECKING_TIMEOUT = 60
+MAX_CLUSTER_WAIT_TIME = 5400
+
+
+class AWSClusterManager:
+    def __init__(self):
+        self.api_client = LightningClient()
+
+    def create(
+        self,
+        cost_savings=None,
+        cluster_name=None,
+        role_arn=None,
+        region=None,
+        external_id=None,
+        instance_types=None,
+        edit_before_creation=None,
+        wait=None,
+    ):
+        performance_profile = V1ClusterPerformanceProfile.DEFAULT
```

Comment:
What are the different options?

Reply:
we only support default and cost savings mode today; it's modeled as an enum in the backend in case we add more modes in the future.